### PR TITLE
[ZH] Update `osu!supporter`

### PR DESCRIPTION
**Pull request by Kotoki1337**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(need review)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/osu!supporter/zh.md
Comment (RushFTK) [reply wiki/osu!supporter/zh.md:123]: 考虑到Disambiguation/Version还没有正式译名，最好看看术语表，天比较晚了这点我就没确认。。。

File: wiki/osu!supporter/zh.md
Comment (RushFTK) [inline wiki/osu!supporter/zh.md:139]:
```suggestion
要购买 osu! 支持者标签，请前往 [osu! 支持者商店页](https://osu.ppy.sh/store/products/supporter-tag)，拖动滑块或点击显示特定月份数的按钮可以调整支持者头衔时长。所有价格都是以美元（USD）为单位，不包括支付系统可能产生的手续费。
```

> using the slider, or the buttons with the number of months underneath.

**你必须修改这条**

**漏翻**点击滑条下面那个 “月 1 2 4”也可以设置撒泼特时长的方式。

File: wiki/osu!supporter/zh.md
Comment (RushFTK) [inline wiki/osu!supporter/zh.md:141]:
```suggestion
之后，点击 `添加到购物车` 将选择好的支持者标签添加到 osu! 商店的购物车。前往 [购物车](https://osu.ppy.sh/store/cart)点击 `结账`，跟随接下来的屏幕上的步骤完成购买。
```

>Afterwards, click Add to Cart to put the supporter tag into the osu!store [shopping cart](https://osu.ppy.sh/store/cart). To finalise the purchase, go to the shopping cart and click Checkout, then follow on-screen prompts.

原文没那么贴心233，不过这里加不加都行

**Review by RushFTK — CHANGES_REQUESTED** [review]
Two of the changes must be fixed or must explain deny reason.
Others are opinion.

File: wiki/osu!supporter/zh.md
Comment (Kotoki1337) [reply wiki/osu!supporter/zh.md:123]: 客户端内“cutting edge”被翻为`测试版`

File: wiki/osu!supporter/zh.md
Comment (Kotoki1337) [reply wiki/osu!supporter/zh.md:141]: 原翻译有这句，我保留了

File: wiki/osu!supporter/zh.md
Comment (Walavouchey) [reply wiki/osu!supporter/zh.md:123]: use what's displayed in-game. it doesn't need quotation marks (`"`) either if it's better without.

File: wiki/osu!supporter/zh.md
Comment (Kotoki1337) [reply wiki/osu!supporter/zh.md:139]: 我觉得这里不应该使用`头衔`，这里购买的是“supporter tag”，而不是单独的一个“title”

File: wiki/osu!supporter/zh.md
Comment (Kotoki1337) [reply wiki/osu!supporter/zh.md:14]: “tag”是“tag”，“title”是“title”，这不是一个概念吧

File: wiki/osu!supporter/zh.md
Comment (Kotoki1337) [reply wiki/osu!supporter/zh.md:46]: 我以为在 web 中 `userpage` 被译为 `玩家信息`，后发现原文不同。我在这篇翻译中尽可能保持了与 osu! 整个项目的本地化字符串对等，所以导致这篇文章看起来有些词并不是很合适比如 `me!` 被译为 `个人介绍`。

will fix this

File: wiki/osu!supporter/zh.md
Comment (RushFTK) [reply wiki/osu!supporter/zh.md:14]: > osu!supporter (or the osu!supporter ***tag***) is a temporary ***title*** given to players who support osu! by making a voluntary, monetary donation to help it thrive. Supporters receive a handful of cosmetic and technical benefits, most of which are active only when the  ***title*** is present. The extra features from osu!supporter do not give any advantage in gameplay, performance, or score, and never will.

原文如上，注意斜体加粗部分。所以本人建议除了括弧里使用"标签"，其余部分替换为"头衔"。

File: wiki/osu!supporter/zh.md
Comment (RushFTK) [reply wiki/osu!supporter/zh.md:46]: 在现在的版本中`me!`确实被用成`个人介绍`，所以咱们的译文可以和个体介绍相同。  
`userpage`翻译为`玩家信息`是我们的字典规范吗，我其实在校对的时候确实没有通读咱们的字典规范，如果你确定的话可以不用强行译为`用户页`。

File: wiki/osu!supporter/zh.md
Comment (RushFTK) [reply wiki/osu!supporter/zh.md:123]: @Kotoki1337 居然现在是这样的，那就还用测试版吧……那这段可以keep，或者你看着语序改?

-----------------------------------

@Walavouchey the English version of this part are follows:
> Supporters may access multiplayer from builds on the early access "cutting edge" release stream.

To Keep insist with [Source File](https://github.com/ppy/osu-wiki/blob/master/wiki/osu!supporter/en.md), I suggest to keep the quotation marks (`"`), if it surely needn't , updated English version first , then fixed on this PR may better.
If necessary, I think translation of a article should give readers reading experience same as "a native speaker reading original article". so "glitch" in original article should also presented on translation version.

File: wiki/osu!supporter/zh.md
Comment (RushFTK) [reply wiki/osu!supporter/zh.md:139]: 头衔可以换成标签，但是按钮的部分必须要加。

File: wiki/osu!supporter/zh.md
Comment (RushFTK) [reply wiki/osu!supporter/zh.md:141]: 现在的原文其实没有了，我的主张主要是基于同步最新版英文原文的原则。如果你确认有必要可以选择keep，我没有额外意见。

**Review by RushFTK — CHANGES_REQUESTED** [review]
Reply code review feedbacks for #7216

File: wiki/osu!supporter/zh.md
Comment (Walavouchey) [reply wiki/osu!supporter/zh.md:123]: The meaning needs to be the same, but not necessarily the formatting. There is nothing wrong with the quotation marks in English, but if they're not needed in Chinese, then it's ok to omit them. Either way, that's your call to make, because I don't know Chinese.

File: wiki/osu!supporter/zh.md
Comment (Kotoki1337) [reply wiki/osu!supporter/zh.md:14]: 上条回复我会错意了，ignore it。不过我不知道为什么原文使用了 title 而不是 tag，因为我翻的时候觉得 title 像是购买 tag 附赠的一个东西，提供功能的是 tag 本身而不是 title，而且游戏中其实也没有实际显示所谓 title 的地方，只有一个 badge，而且网站的 interface 通篇使用的也是 tag 这个词，用 title 可能会给玩家带来误导，不过那是英文的事。
但还是先考虑与英文的对应性，will be fix

File: wiki/osu!supporter/zh.md
Comment (Kotoki1337) [reply wiki/osu!supporter/zh.md:46]: 我也记不清这个词的规范是什么，但非常不幸，名词对照表已经 invalid 了。如果我没能找到其在 osu! 里的对应名词译文，那我会统一成 `用户页`

File: wiki/osu!supporter/zh.md
Comment (Kotoki1337) [reply wiki/osu!supporter/zh.md:14]: `performance` 也许指的是 `performance point`

File: wiki/osu!supporter/zh.md
Comment (Kotoki1337) [reply wiki/osu!supporter/zh.md]: 我开始没能理解具体含义，我清醒之后大概理解了。
但我是比较按照我的思路来的([commit](https://github.com/ppy/osu-wiki/pull/7216/commits/3539f2172afbc94c370ff19256855d4afb928a3d#diff-a1426e80c6184a29fd96465b7791ba053d1ac29d8356ce4f2faacca1c91dfd26R46))，如果你觉得还有问题 just lmk。
先标 resolve 了

File: wiki/osu!supporter/zh.md
Comment (Kotoki1337) [inline wiki/osu!supporter/zh.md:14]: https://discord.com/channels/188630481301012481/218677502141399041/968184608615182398

```suggestion
**osu! 支持者**（osu!supporter，也被称为 ***支持者标签***，玩家相传为 ***撒泼特***）是一个有时效性的头衔，授予那些通过自愿捐款来帮助 osu! 发展的玩家们。支持者们会在标签有效期内获得一些具有装饰性或技术性的额外功能。但无论现在还是将来，osu! 支持者标签的额外功能不会给游戏内容、玩家水平或计分系统带来任何的优势。
```

File: wiki/osu!supporter/zh.md
Comment (RushFTK) [reply wiki/osu!supporter/zh.md:123]: Well, in my opinion, keeping that double quotation is fine. waiting for more reviewer for avoid subjective is also a opinion.

**Review by RushFTK — DISMISSED** [review]
All review are opinions, after all conversion solved, merge could be done.

File: wiki/osu!supporter/zh.md
Comment (Walavouchey) [reply wiki/osu!supporter/zh.md:14]: "Title" here is used in the more general sense of the word (as in "this player is an osu!supporter"). It's not a literal [user title](https://osu.ppy.sh/wiki/en/User_title). The rest of the article refers to it as a tag anyway, so it doesn't need to be translated as a "title" if it causes confusion (and perhaps the English version should be reworded slightly as well, though I don't have any good ideas atm)

Comment (Kotoki1337) [reply wiki/osu!supporter/zh.md:14]: alright, i keep `标签`

File: wiki/osu!supporter/zh.md
Comment (Kotoki1337) [reply wiki/osu!supporter/zh.md:67]: 使用了 `通常情况`，原文那边也只是强调了首次，其余都是在说改名相关的规定。
引用的文章中文翻译也有点误导性，好像是买一次就给一次一样，之后发个pr更新的时候在看吧。
make as resolved

File: wiki/osu!supporter/zh.md
Comment (Kotoki1337) [reply wiki/osu!supporter/zh.md:123]: I chose to keep it, I thought of some solutions for quotation marks but all of them are not compliance with Chinese criteria.

**Review by RushFTK — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**
